randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2678,10 +2685,8 @@ Extra - asset_id: collision_camera_031
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Door to Map Station
       Any of the following:
-          Grapple Beam or Spider Magnet or Space Jump or After Cataris - Lower Lava Button or Infinite Bomb Jump (Beginner)
-          All of the following:
-              Grapple Beam
-              Flash Shift or Use Spin Boost
+          Grapple Beam or Space Jump or Simple IBJ
```

Comment:
can you not get a speed boost on this side?

Reply:
In the room no, will test to see if there is space and a reasonable path there, i believe it would be pretty simple but it would also need an event triggered iirc the north thermal device

Reply:
doesn't need that event, can be done using the area below the room